Makefile

```diff
@@ -182,6 +182,7 @@ apis:
 	@$(ROOT)/scripts/format-json.sh applications/crossbar/priv/couchdb/schemas/*.json
 	@ERL_LIBS=deps/:core/:applications/ $(ROOT)/scripts/generate-api-endpoints.escript
 	@$(ROOT)/scripts/format-json.sh applications/crossbar/priv/api/swagger.json
+	@ERL_LIBS=deps/:core/:applications/ $(ROOT)/scripts/generate-fs-headers-hrl.escript
```

Comment:
@fenollp the expectation is that, when someone adds code to ecallmgr to access a new field in the FS proplist, they will add the field to the .hrl. Then, when the build job runs, the generated hrl will continue to match the committed hrl. If they don't match (just like if there was a formatting change or API doc change) then its on the committer to update the appropriate place. Plus I think the code needs to be compiled prior so the AST can be traversed.